bigquery/nox.py

```diff
@@ -36,7 +36,10 @@ def default(session):
     """
     # Install all test dependencies, then install this package in-place.
     session.install('mock', 'pytest', 'pytest-cov', *LOCAL_DEPS)
-    session.install('-e', '.[pandas]')
+    if session.interpreter == 'python3.4':
```

Comment:
@alixhami I'd rather see you check `py == '3.4'`, but otherwise LGTM. Just merge if you disagree.

Reply:
**UPDATE**: DERP, I now realize why that's not possible. Merge away. Cheers.